### ISSUE TITLE
[Question] How to force select suggestion?

### ISSUE THREAD

**Issue by fabiojansenbr** [AbdulRahmanAlHamali/flutter_typeahead#119]
Hi.

How i can force the user to select some suggestion and not accept itens wich are not on suggestion list? its possible?

thanks

**Comment by hassanbht** [comment]
Hi
i use it in validator

validator: (value) {
               if (_listState != null &&
                  _listState.length > 0 &&
                  _listState.indexWhere((c) => c == value.toString(), 0) < 0) {
                return "you can force to select some suggestion";
              }
              return null;
            },

**Comment by KaYBlitZ** [comment]
Yes, you need to validate the selection if you want to add constraints.
